.github/workflows/pre-merge-checks.yml

```diff
@@ -15,6 +15,10 @@ jobs:
         uses: actions/checkout@v2
         with:
           submodules: true
+      - name: Pre-commit Checks
```

Comment:
The alternative is to use the deprecated, but still functional GHA

https://github.com/pre-commit/action